multi_subject/manual_correction.py

```diff
@@ -0,0 +1,424 @@
+    print("In FSLeyes, click on 'Edit mode', correct the segmentation, then save it with the same name (overwrite).")
+    os.system('fsleyes -yh ' + fname + ' ' + fname_seg_out + ' -cm red')
+
+
+def correct_vertebral_labeling(fname, fname_label):
+    """
+    Open sct_label_utils to manually label vertebral levels.
+    :param fname:
+    :param fname_label:
+    :param name_rater:
+    :return:
+    """
+    message = "Click at the posterior tip of the disc between C1-C2, C2-C3 and C3-C4 vertebral levels, then click 'Save and Quit'."
+    os.system('sct_label_utils -i {} -create-viewer 2,3,4 -o {} -msg "{}"'.format(fname, fname_label, message))
+
+
+def correct_pmj_label(fname, fname_label):
+    """
+    Open sct_label_utils to manually label PMJ.
+    :param fname:
+    :param fname_label:
+    :param name_rater:
+    :return:
+    """
+    message = "Click at the posterior tip of the pontomedullary junction (PMJ) then click 'Save and Quit'."
+    os.system('sct_label_utils -i {} -create-viewer 50 -o {} -msg "{}"'.format(fname, fname_label, message))
+
+
+def create_json(fname_nifti, name_rater):
+    """
+    Create json sidecar with meta information
+    :param fname_nifti: str: File name of the nifti image to associate with the json sidecar
+    :param name_rater: str: Name of the expert rater
+    :return:
+    """
+    metadata = {'Author': name_rater, 'Date': time.strftime('%Y-%m-%d %H:%M:%S')}
+    fname_json = fname_nifti.rstrip('.nii').rstrip('.nii.gz') + '.json'
+    with open(fname_json, 'w') as outfile:
+        json.dump(metadata, outfile, indent=4)
+
+
+def main():
+
+    # Parse the command line arguments
+    parser = get_parser()
+    args = parser.parse_args()
+
+    # Check if required software is installed (skip that if -qc-only is true)
+    if not args.qc_only:
+        if not check_software_installed():
+            sys.exit("Some required software are not installed. Exit program.")
+
+    # check if input yml file exists
+    if os.path.isfile(args.config):
+        fname_yml = args.config
+    else:
+        sys.exit("ERROR: Input yml file {} does not exist or path is wrong.".format(args.config))
+
+    # fetch input yml file as dict
+    with open(fname_yml, 'r') as stream:
+        try:
+            dict_yml = yaml.safe_load(stream)
+        except yaml.YAMLError as exc:
+            print(exc)
+
+    # Curate dict_yml to only have filenames instead of absolute path
+    dict_yml = curate_dict_yml(dict_yml)
+
+    # check for missing files before starting the whole process
+    check_files_exist(dict_yml, args.path_in)
+
+    # check that output folder exists and has write permission
+    path_out_deriv = check_output_folder(args.path_out, FOLDER_DERIVATIVES)
+
+    # Get name of expert rater (skip if -qc-only is true)
+    if not args.qc_only:
+        name_rater = input("Enter your name (Firstname Lastname). It will be used to generate a json sidecar with each "
+                           "corrected file: ")
```

Comment:
When I tried `manual_correction.py`, this step caught me off guard me a bit. During the demo, it might be good to keep this in mind and maybe say "the point of entering your name is to help keep track of different raters if multiple people will be performing manual corrections" or something like that.